.pipelines/load-tests-template.yaml

```diff
@@ -0,0 +1,125 @@
+        displayName: "[AKS] Assign User MSI to the VMSS"
+        condition: and(succeeded(), eq(variables['CLUSTER_TYPE'], 'aks'))
+
+      - script: |
+          echo "Installing helm..."
+          curl https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | bash
+        displayName: "Install helm"
+
+      - script: |
+          kubectl wait --for=condition=ready node --all
+          kubectl wait pod -n kube-system --for=condition=Ready --all
+          kubectl get nodes -owide
+          kubectl cluster-info
+        displayName: "Check cluster's health"
+
+      - script: |
+          export REGISTRY="${REGISTRY:-$(REGISTRY_NAME).azurecr.io/secrets-store-csi-driver-provider-azure}"
+
+          helm install csi manifest_staging/charts/csi-secrets-store-provider-azure --namespace $NAMESPACE --wait --timeout=10m -v=5 --debug \
+            --set image.repository=${REGISTRY} \
+            --set image.tag=${IMAGE_VERSION} \
+            --dependency-update
+
+          envsubst < test/load/config-deployment-template.yaml > test/load/config.yaml
+          envsubst < test/load/deployment-template.yaml > test/load/deployment.yaml
+          envsubst < test/load/secret-provider-class-template.yaml > test/load/secret-provider-class.yaml
+        displayName: "Deploy secrets-store-csi-driver and azure keyvault provider"
+        workingDirectory: "$(system.defaultWorkingDirectory)"
+        env:
+          NAMESPACE: "kube-system"
+          SUBSCRIPTION_ID: $(LOAD_TEST_SUBSCRIPTION_ID)
+          USER_ASSIGNED_IDENTITY_CLIENT_ID: $(USER_ASSIGNED_IDENTITY_CLIENT_ID)
+
+      - script: |
+          echo "--- CONFIG ---"
+          cat test/load/config.yaml
+          echo "--- DEPLOYMENT ---"
+          cat test/load/deployment.yaml
+          echo "--- SECRET PROVIDER CLASS ---"
+          cat test/load/secret-provider-class.yaml
+        workingDirectory: "$(system.defaultWorkingDirectory)"
+        displayName: "Get yaml"
+
+      - script: |
+          let NODE_COUNT=${AGENT_COUNT}*${NODE_POOL_COUNT}
+          ./run-e2e.sh cluster-loader2 \
+            --nodes=${NODE_COUNT} \
+            --provider=local \
+            --report-dir="$(perf-tests.repo.path)/_artifacts" \
+            --masterip=${MASTERIP} \
+            --master-internal-ip=${MASTERINTERNALIP} \
+            --testconfig="$(system.defaultWorkingDirectory)/test/load/config.yaml"
+        workingDirectory: "$(perf-tests.repo.path)"
+        displayName: "Run clusterloader2 test"
+
+      - script: |
+          echo "--- GET NODES ---"
+          kubectl get nodes
+          echo "--- GET NS ---"
+          kubectl get ns
+          echo "--- GET PODS ---"
+          kubectl get pods --all-namespaces
+          echo "--- TOP PODS ---"
+          kubectl top pods -n kube-system
+        workingDirectory: "$(perf-tests.repo.path)"
```

Comment:
would it make sense to list the SecretProviderClass objects?

Reply:
The `SecretProviderClass` is only user created and doesn't contain any other metadata. So I think we don't need to list the objects. We do print the template after substituting values - https://github.com/Azure/secrets-store-csi-driver-provider-azure/pull/219/files#diff-27c0385b26ddad592b2fdf50544d7e0eR74 which should be sufficient.